ush/python/pygw/src/schema_pkg/test_files/model_configure.schema.yaml

```diff
@@ -0,0 +1,70 @@
+required:
```

Comment:
These are good tests.
My only suggestion would be to turn these into pytest fixtures with a more variety of type checking.
There is no need to have a test with 10+ `int` checks and not a single `float` or `complex` or `array` etc.